platform-tests/src/test/java/org/junit/platform/commons/util/ReflectionUtilsTests.java

```diff
@@ -368,99 +408,102 @@ void invokePrivateStaticMethod() throws Exception {
 	}
 
 	@Test
-	void loadClassPreconditions() {
```

Comment:
I might have overlooked something, but it appears that you have removed tests for deprecated utility methods. If that's the case, I'd recommend leaving such tests in place until the deprecated code has been officially removed.

Reply:
Unless I have overlooked something, I've kept around a sufficient amount of tests for each deprecated method to make sure they delegate to their `try...` variant correctly but have deleted any surplus tests for special cases already tested for the `try...` variant. Makes sense?

Reply:
Yep, if that's the case... that makes sense.